config/notifiers.go

```diff
@@ -250,11 +254,47 @@ func (c *SlackAction) UnmarshalYAML(unmarshal func(interface{}) error) error {
 		return fmt.Errorf("missing type in Slack action configuration")
 	}
 	if c.Text == "" {
-		return fmt.Errorf("missing value in Slack text configuration")
+		return fmt.Errorf("missing text value in Slack text configuration")
 	}
-	if c.URL == "" {
-		return fmt.Errorf("missing value in Slack url configuration")
+	// either URL or Name and Value need to be provided
+	if c.URL == "" && c.Name == "" {
+		return fmt.Errorf("missing name or url in Slack action configuration")
 	}
+	if (c.Name == "" || c.Value == "") && c.URL == "" {
```

Comment:
I'm still not sure that it is the correct approach: AFAIU `value` isn't mandatory. Also what happens if the configuration defines `url` and `name` at the same time? What is the result in Slack?

Maybe do something like this instead?

```go
if c.URL != "" {
  // Clear all message action fields.
  c.Name = ""
  c.Value = ""
  c.ConfirmField = nil
} else if c.Name != "" {
  c.URL = ""
} else {
  return fmt.Errorf("missing name or url in Slack action configuration")
}
```